docs/source/workflows/editor-plugins.md

```diff
@@ -1,65 +1,61 @@
 ---
-title: Connecting Apollo to your editor
+title: Editor plugins
 description: How to get the most out of your editor with Apollo
 ---
 
 GraphQL has the potential to create incredible developer experiences, thanks to its strongly typed schema and query language. The Apollo platform brings these possibilities to life by enhancing your editor with rich metadata from your graph API. Currently only [Visual Studio Code](https://code.visualstudio.com/) (VS Code) is supported, but more are coming soon.
 
 <img src="../images/editors/jump-to-def.gif" width="80%" style="margin: 5%" alt="Using jump to definition on a fragment">
 
-## Apollo VS Code
+<h2 id="vscode">Apollo VS Code</h2>
```

Comment:
I think we should stay consistent with the rest of the docs and use `##`  here. What do you think?